deployment/IMPORT_INSTRUCTIONS.md

```diff
@@ -0,0 +1,35 @@
+# Import Instructions
```

Comment:
@JonHolman This is a much more stripped down set of instructions like you suggested.